src/Manager/Data/Manager.php

```diff
@@ -20,14 +19,19 @@
 
 class Manager
 {
+    /** @var GetId */
```

Comment:
I got so into removing all my superfluous comments that I forgot to include these types for object properties. Kinda jumped right to the other side of the spectrum haha.

Reply:
I honestly thought you were doing this so a conversion to types properties would be easier in PHP 7.4 (https://wiki.php.net/rfc/typed_properties_v2)

I like the idea of not having them, it's just a little easier in the meantime for IDE's